compat-28.el

```diff
@@ -392,6 +392,26 @@ not a list, return a one-element list containing OBJECT."
       object
     (list object)))
 
+(compat-defun get-buffer-create (name &optional inhibit-buffer-hooks) ;; <compat-tests:buffer-tests-inhibit-buffer-hooks>
```

Comment:
Our convention is to have separate tests for `get-buffer-create` and `generate-new-buffer`. They should be named `compat-get-buffer-create` and `compat-generate-new-buffer`.